pkg/controllers/provisioning/scheduling/nodeclaim.go

```diff
@@ -419,14 +420,18 @@ func filterInstanceTypesByRequirements(instanceTypes []*cloudprovider.InstanceTy
 		// We don't care about the minimum number of instance types that meet our requirements here, we only care if they meet our requirements.
 		_, err.minValuesIncompatibleErr = remaining.SatisfiesMinValues(requirements)
```

Comment:
We should perhaps consider returning a modified requirement set with modified minValues based on the result of the relaxation and the remaining instance types attached to the NodeClaim